src/Microsoft.Framework.Configuration/ConfigurationBase.cs

```diff
@@ -86,12 +86,20 @@ public IConfigurationSection GetSection(string key)
         {
             if (string.IsNullOrEmpty(key))
             {
-                throw new InvalidOperationException(Resources.Error_EmptyKey);
+                throw new ArgumentException(Resources.Error_EmptyKey);
             }
 
-            return new ConfigurationSection(Sources, GetPrefix() + key);
+            return new ConfigurationSection(Sources, GetPath(), key);
         }
 
-        protected abstract string GetPrefix();
+        private string GetPath()
```

Comment:
Anyway, maybe I am missing something. I will setup something so that we can review this in person.